### PR TITLE
JBPM-5536: Lienzo Panel & implementations - Tests

### PR DESCRIPTION
**Pull request by romartin**
Hey @hasys 

These are the fixes for making all tests work (regression), still working on the new ones, sorry will provide ASAP, but at least you can start the review.

Depends on https://github.com/kiegroup/lienzo-core/pull/70

Thanks!

### CODE REVIEW

File: src/test/java/com/ait/lienzo/client/widget/panel/impl/PreviewLayerTest.java [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+{
+    private static final Bounds BG_BOUNDS      = Bounds.build(0d, 0d, 800d, 1200d);
+
+    private static final Bounds VISIBLE_BOUNDS = Bounds.build(0d, 0d, 234.3d, 654.5d);
+
+    private PreviewLayer     tested;
+
+    private Supplier<Bounds> bgBoundsSupplier;
+
+    private Supplier<Bounds> clearBoundsSupplier;
```
Comment (hasys) [inline src/test/java/com/ait/lienzo/client/widget/panel/impl/PreviewLayerTest.java]: Field can be converted to a local variable

File: src/test/java/com/ait/lienzo/client/widget/panel/scrollbars/ScrollUITest.java [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+import static org.mockito.Mockito.doReturn;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.spy;
+import static org.mockito.Mockito.verify;
+
+@RunWith(LienzoMockitoTestRunner.class)
+public class ScrollUITest {
+
+    @Mock
+    private AbsolutePanel panel;
```
Comment (hasys) [inline src/test/java/com/ait/lienzo/client/widget/panel/scrollbars/ScrollUITest.java]: Private field 'panel' is never used

File: src/test/java/com/ait/lienzo/client/widget/panel/scrollbars/ScrollBoundsTest.java [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+    public void testMaxBoundX() {
+        assertEquals(248.1d, tested.maxBoundX(), 0);
+    }
+
+    @Test
+    public void testMaxBoundY() {
+        assertEquals(920.5d, tested.maxBoundY(), 0);
+    }
+
+    // TODO: Review result = 0
```
Comment (hasys) [inline src/test/java/com/ait/lienzo/client/widget/panel/scrollbars/ScrollBoundsTest.java]: TODO

File: src/test/java/com/ait/lienzo/client/widget/panel/scrollbars/ScrollBoundsTest.java [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+    @Test
+    public void testMinBoundX() {
+        assertEquals(0d, tested.minBoundX(), 0);
+    }
+
+    // TODO: Review result = 0
```
Comment (hasys) [inline src/test/java/com/ait/lienzo/client/widget/panel/scrollbars/ScrollBoundsTest.java]: todo

File: src/test/java/com/ait/lienzo/client/widget/panel/scrollbars/ScrollBoundsTest.java (hunk 3 above)
Comment (romartin) [reply src/test/java/com/ait/lienzo/client/widget/panel/scrollbars/ScrollBoundsTest.java]: done

File: src/test/java/com/ait/lienzo/client/widget/panel/scrollbars/ScrollBoundsTest.java (hunk 4 above)
Comment (romartin) [reply src/test/java/com/ait/lienzo/client/widget/panel/scrollbars/ScrollBoundsTest.java]: done

**Comment by romartin** [comment]
squashed commits manually before merging

**Pull request merged**